modules/icingadirector/collector.go

```diff
@@ -54,14 +54,16 @@ func Collect(c *collection.Collection) {
 		c.AddFilesIfFound(ModuleName, file)
 	}
 
-	for name, element := range journalctlLogs {
-		if service, err := collection.FindServices(element.Service); err == nil && len(service) > 0 {
-			c.AddCommandOutput(filepath.Join(ModuleName, name), "journalctl", "-u", element.Service, "--since \"7 days ago\"")
-		}
-	}
-
 	// Get GIT Repository details
 	if path, ok := collection.IsGitRepository(InstallationPath); collection.DetectGitInstalled() && ok {
 		c.AddGitRepoInfo(filepath.Join(ModuleName, "git-info.yml"), path)
 	}
+
+	if c.Detailed {
+		for name, element := range journalctlLogs {
+			if service, err := collection.FindServices(element.Service); err == nil && len(service) > 0 {
+				c.AddCommandOutput(filepath.Join(ModuleName, name), "journalctl", "-u", element.Service, "--since \"7 days ago\"")
```

Comment:
Why not `, "--since", "7 days ago")` like above ?

Reply:
Done with 8b74214.